recipes/snappy/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version = "1.1.3" %}
+
+package:
+  name: snappy
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/google/snappy.git
```

Comment:
Could we just download the [tarball]( https://github.com/google/snappy/releases/download/1.1.3/snappy-1.1.3.tar.gz ) instead? Should be faster and may avoid a few extra configuration steps.